File: app/src/main/java/me/jojigarcia/impuestoscalculator/MainActivity.java
package me.jojigarcia.impuestoscalculator;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    private Button btCalculateIVA, btSubIVA;
    private EditText conceptoField;
    private TextView moneyText;
    private int saldo;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btCalculateIVA = (Button) findViewById(R.id.btCalculateIVA);
        btSubIVA = (Button) findViewById(R.id.btSubIVA);
        conceptoField = (EditText) findViewById(R.id.conceptoField);
        moneyText = (TextView) findViewById(R.id.moneyText);

        moneyText.setText("0.0€");

        btCalculateIVA.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                saldo = Integer
            }
        });
    }
}

File: app/src/main/java/me/jojigarcia/impuestoscalculator/MainActivity.kt
package me.jojigarcia.impuestoscalculator

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    private var btCalculateIVA: Button? = null
    private var btSubIVA: Button? = null
    private var conceptoField: EditText? = null
    private var moneyText: TextView? = null
    private var saldo: Double = 0.0
    private var saldoIntroducido: Double = 0.0
    private var IVA: Double = 0.79

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btCalculateIVA = findViewById(R.id.btCalculateIVA) as Button
        btSubIVA = findViewById(R.id.btSubIVA) as Button
        conceptoField = findViewById(R.id.conceptoField) as EditText
        moneyText = findViewById(R.id.moneyText) as TextView

        moneyText!!.text = "0.0€"
        saldoIntroducido = java.lang.Double.parseDouble(conceptoField.toString())

        btCalculateIVA!!.setOnClickListener {
            saldo = saldoIntroducido / IVA
            moneyText!!.text = saldo.toString()
        }
    }
}
